components/ArticlePreview/ArticlePreview.tsx

```diff
@@ -160,20 +160,20 @@ const ArticlePreview: NextPage<Props> = ({
                   leaveFrom="transform opacity-100 scale-100"
                   leaveTo="transform opacity-0 scale-95"
                 >
-                  <Menu.Items className="origin-top-right absolute bottom-10 right-0 mt-2 w-48 rounded-md shadow-lg py-1 bg-white ring-1 px-1 ring-black ring-opacity-5 focus:outline-none">
+                  <Menu.Items className="origin-top-right absolute bottom-10 right-0 mt-2 w-48 rounded-md shadow-lg py-1 bg-black ring-white dark:bg-white ring-1 px-1 dark:ring-black ring-opacity-5 focus:outline-none">
```

Comment:
Could you reverse the styles and screenshot what it looks like? 
I didn't even think about the dark mode for the menu options. 🧠 

It looks good but I think it makes more sense to have a dark manu on the dark mode (and not the light mode). Thanks! 🙌

Reply:
@NiallJoeMaher It looks like this.

![image](https://github.com/codu-code/codu/assets/109150688/b3cea7fe-e369-4cb0-9d98-f020f5e374ad)

Reply:
This looks great! Let's get that pushed and I'll approve.